.github/workflows/release-6.4.yml

```
@@ -99,8 +99,8 @@ jobs:
         - name: Install Node and dependencies
           uses: mongodb-labs/drivers-github-tools/node/setup@v2
   
-        # - run: npm publish --provenance --tag=latest
-        #   if: ${{ needs.release_please.outputs.release_created }}
-        #   env:
-        #     NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        - run: npm publish --provenance --tag 6.4 --dry-run
```

```suggestion
      - run: npm publish --provenance --tag prev --dry-run
```

Anything will do here: "prev"/"legacy"/"tag-for-publishing-older-releases". But it cannot be something that is ambiguous with a version number or a semver range, npm will error.